CHANGELOG.md

```diff
@@ -21,7 +21,10 @@
 
 #### Bug Fixes
 
-* None.
+* Fixed `vertical_whitespace_between_cases` rule containing strings.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

Reply:
@marcelofabri: fixed.
As far as I understand, it is OK that some checks failed at https://github.com/realm/SwiftLint/pull/2865#issuecomment-528404066 ? The check assumes that this rule is turned on while the code is written without such assumption.

Reply:
It's expected - they serve as a sanity check so we can manually inspect them and see if we're introducing any false positives/negatives

Reply:
@marcelofabri: fixed trailing spaces, is there any other problems with the code?

Reply:
I think this would introduce false positives on strings (which we might be ok with since this is opt-in). Can you add a test that contains a string a switch as a string?

Reply:
@marcelofabri: Added a test example here: https://github.com/realm/SwiftLint/pull/2865/commits/90a235971393b2d06f603be47780ac05e5a7446a. Tests are running correctly. Is this test ok?

Reply:
@marcelofabri: any updates about this issue?